x-pack/plugins/lens/public/indexpattern_datasource/loader.ts

```diff
@@ -92,7 +93,7 @@ export function convertDataViewIntoLensIndexPattern(dataView: DataView): IndexPa
       Object.fromEntries(
         Object.entries(fieldFormatMap).map(([id, format]) => [
           id,
-          'toJSON' in format ? format.toJSON() : format,
+          'toJSON' in format ? (format as unknown as FieldFormat).toJSON() : format, // FIXME SerializedFieldFormat was inferred... was this intended to work with FieldFormat instead?
```

Comment:
I need reviewers to look closely at this: I could not see how there would be a `.toJSON` property on this object.